### ISSUE TITLE
ValueError: max() arg is an empty sequence - after upgrade to 2024.8.1

### ISSUE THREAD

**Issue by jeremysherriff** [bkbilly/lnxlink#122]
### What type of installation are you running?

Server

### Which Linux OS are you using?

Ubuntu 22.04

### Which version of LNXLink has the issue?

2024.8.1

### Describe the problem

Error message in logs upon service start:
```
2024-08-09 16:53:01,600 [9181:MainThread.__main__.__init__.46] [INFO]  LNXlink 2024.8.1, Python 3.10.12
2024-08-09 16:53:01,700 [9181:MainThread.__main__.start.84] [INFO]  Loaded addons: inference_time, bash, mounts, sys_updates, required_restart, memory, restart, statistics, update, cpu
2024-08-09 16:53:01,708 [9181:Thread-2 (_thread_main).__main__.on_connect.241] [INFO]  MQTT connection: Connection Accepted.
2024-08-09 16:53:01,763 [9181:Thread-2 (_thread_main).__main__.run_module.135] [ERROR]  Error with addon Inference Time: max() arg is an empty sequence, Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/lnxlink/__main__.py", line 130, in run_module
    pub_data = method()
  File "/usr/local/lib/python3.10/dist-packages/lnxlink/modules/inference_time.py", line 17, in get_info
    "max": max(
ValueError: max() arg is an empty sequence
```

### Additional context

Error was not present in 2024.7.0

### Logs

```bash
Aug 09 16:53:01 stuff lnxlink[922858]: INFO:lnxlink:LNXlink 2024.8.1, Python 3.10.12
Aug 09 16:53:01 stuff lnxlink[922858]: INFO:dasbus.connection:Connecting to the system bus.
Aug 09 16:53:01 stuff lnxlink[922858]: INFO:dasbus.connection:Connecting to the system bus.
Aug 09 16:53:01 stuff lnxlink[922858]: INFO:lnxlink:Loaded addons: inference_time, bash, mounts, sys_updates, required_restart, memory, restart, statistics, update, cpu
Aug 09 16:53:01 stuff lnxlink[922858]: INFO:lnxlink:MQTT connection: Connection Accepted.
Aug 09 16:53:01 stuff lnxlink[922858]: ERROR:lnxlink:Error with addon Inference Time: max() arg is an empty sequence, Traceback (most recent call last):
Aug 09 16:53:01 stuff lnxlink[922858]:   File "/usr/local/lib/python3.10/dist-packages/lnxlink/__main__.py", line 130, in run_module
Aug 09 16:53:01 stuff lnxlink[922858]:     pub_data = method()
Aug 09 16:53:01 stuff lnxlink[922858]:   File "/usr/local/lib/python3.10/dist-packages/lnxlink/modules/inference_time.py", line 17, in get_info
Aug 09 16:53:01 stuff lnxlink[922858]:     "max": max(
Aug 09 16:53:01 stuff lnxlink[922858]: ValueError: max() arg is an empty sequence
Aug 09 17:08:03 stuff lnxlink[922858]: INFO:lnxlink:Sending statistics data: {"uuid": "xxxxxxxxxxxxxxxxxxxxxx", "version": "2024.8.1"}
```

**Comment by jeremysherriff** [comment]
Inference data appears to still be sent to mqtt fine.

**Comment by bkbilly** [comment]
This should only show once it starts and doesn't affect the normal operation of the app.
I found the issue and it should be fixed on the next release.

**Comment by jeremysherriff** [comment]
Yeah confirmed, it's only logged once and no problems with the data.

**Comment by bkbilly** [comment]
I've released the 2024.9.0 version of LNXlink which fixes this issue.
